lib/object-cache.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+const NodeCache = require('node-cache')
+var hash = require('object-hash')
+
+/** ObjectCache class. */
+class ObjectCache {
+  /**
+   * Create Object Cache
+   * @param {number} backendCacheDurationSeconds  - Duration in seconds to save keys in the cache
+   */
+  constructor (backendCacheDurationSeconds) {
+    this._cache = new NodeCache({ stdTTL: backendCacheDurationSeconds / 1000 })
+  }
+
+  /**
+   * Set object in the cache
+   */
+  set (object) {
+    const objectHash = hash(object)
+    this._cache.set(objectHash)
+  }
```

Comment:
Doesn't actually set anything in the cache? 
https://github.com/node-cache/node-cache#store-a-key-set

```suggestion
  set (object, value) {
    const objectHash = hash(object)
    this._cache.set(objectHash, value)
  }
```